react-game/react-game/src/Components/Cards.js

```diff
@@ -0,0 +1,176 @@
+import React from 'react';
+
+export class Cards extends React.Component {
+ constructor(props) {
+  super(props);
+  this.state = JSON.parse(localStorage.getItem('state_cards')) ? JSON.parse(localStorage.getItem('state_cards')) : {array: this.props.cardsarray, cards1: [], cards2: [], cards3: [], cards4: [], trash: [], style1: '0.5rem #00ff4c solid', style2: '0.5rem #00ff4c solid', style3: '0.5rem #00ff4c solid', style4: '0.5rem #00ff4c solid', cont1: 0, cont2: 0,cont3: 0,cont4: 0,a1:false, pressKey: false, volume: this.props.volume()/100}
+   this.setState({style1: '0.5rem #00ff4c solid',style2: '0.5rem #00ff4c solid',style3: '0.5rem #00ff4c solid',style4: '0.5rem #00ff4c solid'});
+   setInterval(() => {
+      this.checkNewGame.call(this);
+   }, 1000);
+}
+
+_handleKeyDown = (event) => {
+   if (!event.repeat) {
+      switch( event.key) {

```

Comment:
to simplify:
case '1', '2', '3', '4':
 this.deleteCard.call(this, {target:{id:`cards${event}`}});

or even to use if condition